diagnostic.md

```diff
@@ -15,23 +15,37 @@ Place your responses inside the fenced code-blocks where indivated by comments.
   join table with references to `Movies` and `Profiles`.
 
   ```md
-    # < Your Response Here >
+  profiles has many movies through favorites. And movies have many profiles through favorites. Favorites belongs to profiles.
   ```
 
 1.  For the above example, what needs to be added to the Model files?
 
   ```rb
   class Profile < ActiveRecord::Base
```

Comment:
You need colons in the proper places here to denote the names of the options and the symbols.

```ruby
class Profile < ActiveRecord::Base
  has_many :movies, through: :favorites
  has_many :favorites
end
```